test/extended/kernel/tools.go

```diff
@@ -185,3 +188,54 @@ func getProcessorCount(oc *exutil.CLI) (int, error) {
 
 	return count, nil
 }
+
+type rtevalOutput struct {
+	XMLName    xml.Name `xml:"rteval"`
+	Statistics struct {
+		XMLName               xml.Name `xml:"statistics"`
+		Samples               int      `xml:"samples"`
+		Minimum               int      `xml:"minimum"`
+		Maximum               int      `xml:"maximum"`
+		Median                float32  `xml:"median"`
+		Mode                  int      `xml:"mode"`
+		Range                 int      `xml:"range"`
+		Mean                  float32  `xml:"mean"`
+		MeanAbsoluteDeviation float32  `xml:"mean_absolute_deviation"`
+		StandardDeviation     float32  `xml:"standard_deviation"`
+	} `xml:"Measurements>Profile>cyclictest>system>statistics"`
+}
+
+func runRteval(oc *exutil.CLI) error {
+	// Create a working directory
+	rtevalWorkDir := "/tmp"
```

Comment:
Small nit, using `os.TempDir()` would be better here to avoid someone else doing things in that dir.

Reply:
I'll update this comment, os.TempDir() would create a temp dir on the machine running the tests, not the machine the tests are being run against

Reply:
Thanks! I missed that part, lgtm to me otherwise.